romancal/resample/meta_blender.py

```diff
@@ -0,0 +1,66 @@
+import numpy as np
+from roman_datamodels import datamodels, maker_utils
+
+
+class MetaBlender:
+    def _blend_first(self, model):
+        # make a blank mosic metdata node
+        self._model = maker_utils.mk_datamodel(
+            datamodels.MosaicModel, n_images=0, shape=(0, 0)
+        )
+        self._meta = self._model.meta
+
+        self._model["individual_image_cal_logs"] = []
+
+        self._mid_mjds = []
+
+        self._meta.basic.time_first_mjd = model.meta.exposure.start_time.mjd
+        self._meta.basic.time_last_mjd = model.meta.exposure.end_time.mjd
+
+        # copy some metadata from the first input model
+        # FIXME this is incorrect and these values should be
```

Comment:
This reproduces the current behavior on main. See https://github.com/spacetelescope/romancal/issues/1598